services/categories.js

```diff
@@ -27,6 +27,17 @@ const listCategories = async () => {
   }
 }
 
+const upDateCategory = async (categoryToEdit, id) => {
+  try {
+    const editCategory = await Category.update(categoryToEdit, {
+      where: { id },
+    })
+    return editCategory[0] === 1 ? { code: 200, status: true, message: 'Category edited' } : { code: 400, status: false, message: `Category ${id} not found` }
```

Comment:
Un detalle debería ser código 404 Not found para normalizar con otras respuestas.